overlay/etc/chromium/master_preferences

```diff
@@ -1,26 +0,0 @@
-{
```

Comment:
Cool thanks. We'll leave it out for now and look into fixing it. Thanks for the links.